### PR TITLE
 📚 Update README.md

### PR DESCRIPTION
**Pull request by elisabettai**
Added link to webinar

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]

**Comment by elisabettai** [comment]
Hi @pcrespov, I see that the CI is failing with some issues regarding `'jinja2_time'`. I saw that locally when using the cookiecutter some weeks ago as well.
Let me know if you know how to solve, otherwise I can also take a look.

**Comment by pcrespov** [comment]
> Hi @pcrespov, I see that the CI is failing with some issues regarding `'jinja2_time'`. fyi: I saw that locally when using the cookiecutter some weeks ago as well. Let me know if you know how to solve, otherwise I can also take a look.

@elisabettai  the CI in https://github.com/ITISFoundation/cookiecutter-osparc-service/actions/runs/5997638565/job/16264399226#step:6:241 shows that jinja2 is installed. My guess is that the version is not pinned and a new upgrade happen in which they removed the dependency to `jinja_time`. It might be that you need to add an extra as `jinja2[time]` ... not sure. 
If this is the case, solutions might go by pin `jinja2` to an older version or add the missing library in the requirements
Can you please investigate further and take this issue?

**Review by pcrespov — COMMENTED** [review]
Please, do not forget to fill the attributes of the PR (.e.g assignee, milestone, etc) so we can track them properly . thx :-)

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ cookiecutter gh:ITISFoundation/cookiecutter-osparc-service
 
 Check [FAQ](./FAQ.md)
 
+## Tutorials and guides
+A video tutorial is available [here](https://youtu.be/yoKXHMQcs1Y?si=lmJgDM-Y52jvwS3d).
```
Comment (pcrespov) [inline README.md:34]: You could do something more "fancy" thatn "here" and embedd the video?

SEE https://www.darenmay.com/custommayd/developer/training/videos/markdown/2020/06/23/embed-youtube-video-in-markdown.html

**Pull request merged**